components/popover/popover-mixin.js

```diff
@@ -100,19 +106,41 @@ export const PopoverMixin = superclass => class extends superclass {
 			this._previousFocusableAncestor = this.opened ? getPreviousFocusableAncestor(this, false, false) : null;
 
 			if (this.opened) {
+
 				this._opener = getComposedActiveElement();
 				this._addAutoCloseHandlers();
-				this._dismissibleId = setDismissible(() => this._close());
+				this._dismissibleId = setDismissible(() => this.close());
+				this._focusContent(this);
 				this.dispatchEvent(new CustomEvent('d2l-popover-open', { bubbles: true, composed: true }));
+
 			} else if (changedProperties.get('opened') !== undefined) {
+
 				this._removeAutoCloseHandlers();
 				this._clearDismissible();
+				this._focusOpener();
 				this.dispatchEvent(new CustomEvent('d2l-popover-close', { bubbles: true, composed: true }));
+
 			}
 
 		}
 	}
 
+	close() {
+		this.opened = false;
+		return this.updateComplete;
+	}
+
+	open(applyFocus = true) {
+		this._applyFocus = applyFocus !== undefined ? applyFocus : true;
```

Comment:
I think this `_applyFocus` semi-state variable could cause some issues if `open()` is called in one place (which would set it) and then the `opened` property or attribute are manipulated?

Reply:
I agree there is some unwanted complexity here, which I am still hoping to remove in my next PR. I'd actually started down the path of introducing a `configure(...)` method you'd mentioned during the previous demo, but then ran into some other awkwardness which caused me to backtrack. I was thinking about it some more last night and convinced myself to try it again. Long story short, I am hoping to remove the `opened` public property from the mixin in another PR.